obal/data/roles/build_package/tasks/wait.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: 'Get {{ build_package_koji_command }} task ids'
+  set_fact:
+    build_package_koji_tasks: "{{ build_package_tito_release.stdout_lines \
+      | select('match', '^Created task:.*') \
+      | map('regex_replace', '^Created task:\\s(\\d+)', '\\1') \
+      | list }}"
+  when: normal_execution_check is not defined
```

Comment:
I don't see this defined anywhere

Reply:
Yeah, it got removed.   I think i didn't understand at that moment what it actually did, but now that I've played with these paths so much it insured setting of --tags wait when you wanted to wait instead of doing it by default.  Should I put 'normal_execution_check' back instead of requiring '--skip-tags wait' ??